tests/test_logger.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+#
+# Copyright (c) 2021 Authors and contributors
+#
+# Released under the GNU Public Licence, v2 or any higher version
+# SPDX-License-Identifier: GPL-2.0-or-later
+"""Test mdacli logger."""
+import logging
+
+import mdacli.logger
+
+
+class Test_setup_logger:
+    """Test the setup_logger."""
+
+    def log_msg(self):
+        """Info message parsed to the log."""
+        logger = logging.getLogger("test")
+        logger.info("foo")
+
+    def test_default_log(self, caplog):
+        """Default message in STDOUT."""
+        caplog.set_level(logging.INFO)
+        with mdacli.logger.setup_logging(logfile=None, debug=False):
```

Comment:
very nice work. I never tested log myself. :clap: :clap: this is getting very high level here!!
Again my :exploding_head: with `caplog`. Where does it comes from? :rofl: omg... my brain is getting too much functionalized "all variables are params" - kind of thing hehe :smiling_imp: